authentication/auth.go

```diff
@@ -245,12 +218,32 @@ func generateClaimsV2(claimValues map[string][]string, launcherSchema surveys.La
 	surveyMetadata := make(map[string]interface{})
 	data := make(map[string]interface{})
 
-	if schema.SurveyType == "social" || schema.SurveyType == "health" {
+	if isNonBusinessNonDefaultSurvey(schema.SurveyType) {
```

Comment:
Rather than NonBusiness we can just use Social, but what is meant by NonDefault?

Reply:
When I created this method, I thought it would be suitable as NonBusiness as the method checks if the type is `social`, `health` or `ukhsa-ons`. I thought NonDefault meant anything that wasn't considered the three I mentioned.

Reply:
Those types are all classed as Social Surveys though (even Health) in ONS terms.

Reply:
Updated 👍